src/App.js

```diff
@@ -1,10 +1,24 @@
 import React, { PureComponent } from 'react';
-import './App.css';
+import './assets/App.css';
+import { Routes, Route } from 'react-router-dom';
 import Calculator from './components/Calculator';
+import Header from './components/Header';
+import Quote from './components/Quote';
+import Home from './components/Home';
 
 class App extends PureComponent {
   render() {
-    return <Calculator />;
+    return (
+      <>
+        <Header />
+        <Routes className="content">
+          <Route path="/" element={<Home className="content" />} />
+          <Route path="calculator" element={<Calculator />} />
+          <Route path="quote" element={<Quote />} />
+
+        </Routes>
```

Comment:
- [ ] It will be nice to set the home as the default page when the app starts. For some reason, it's not working like that in your project. I have to click on the home navigation before the component got rendered. I will suggest you take a look at it.

See the test demo to understand what I'm pointing out here.
Demo|
:--------:|
![a1](https://user-images.githubusercontent.com/39523312/160676139-16bb0982-a2b6-4b63-8d9f-5aef4681464f.gif)

Reply:
okay you right , I did fix it